group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorService.java

```diff
@@ -148,7 +157,9 @@ public GroupCoordinatorService build() {
             CoordinatorEventProcessor processor = new MultiThreadedEventProcessor(
                 logContext,
                 "group-coordinator-event-processor-",
-                config.numThreads
+                config.numThreads,
+                coordinatorRuntimeMetrics,
```

Comment:
nit: there are some places in this file where we specify the time before the metrics. Should we try to keep ordering consistent?